libraries/AP_RangeFinder/RangeFinder.cpp

```diff
@@ -36,7 +36,7 @@ const AP_Param::GroupInfo RangeFinder::var_info[] = {
     // @Param: _TYPE
     // @DisplayName: Rangefinder type
     // @Description: What type of rangefinder device that is connected
-    // @Values: 0:None,1:Analog,2:MaxbotixI2C,3:LidarLiteV2,5:PX4-PWM,6:BBB-PRU,7:LightWareI2C,8:LightWareSerial,9:Bebop,10:MAVLink,11:uLanding,12:LeddarOne,13:MaxbotixSerial,14:TrOneI2C,15:LidarLiteV3
+    // @Values: 0:None,1:Analog,2:MaxbotixI2C,3:LidarLiteI2CV1&2,5:PX4-PWM,6:BBB-PRU,7:LightWareI2C,8:LightWareSerial,9:Bebop,10:MAVLink,11:uLanding,12:LeddarOne,13:MaxbotixSerial,14:TrOneI2C,15:LidarLiteI2CV3
```

Comment:
I agree with Tom, lowercase v looks better. I would also use a slash instead of the ampersand (and optionally another v): LidarLiteI2Cv1/v2

In any case I think no one is really fussed about this, so up to you @gmorph!